tests/puppeteer/campaigns/sanity/ordersBO/01_filterOrders.js

```diff
@@ -43,13 +43,13 @@ global.scenario('Filter the Orders table by ID, REFERENCE, STATUS', async () =>
   test('should filter the Orders table by ID and check the result', async () => {
     await orderPage.filterTableByInput(orderPage.orderFilterIdInput, '1', orderPage.searchButton);
     await boBasePage.checkTextValue(orderPage.orderfirstLineIdTD, '1');
-    await orderPage.waitForSelectorAndClick(orderPage.resetButton)
+    await orderPage.waitForSelectorAndClick(orderPage.resetButton);
   });
 
   test('should filter the Orders table by REFERENCE and check the result', async () => {
     await orderPage.filterTableByInput(orderPage.orderFilterReferenceInput, 'FFATNOMMJ', orderPage.searchButton);
```

Comment:
Is there maybe another way to find a product, less tied to fixtures ?

Reply:
You mean order ?
maybe improve test by using an data/order.js file
But not linked this PR but to this one #15312